packages/react/src/components/MDXComponents/components/_utils.scss

```
@@ -0,0 +1,18 @@
+/*
+ * Copyright IBM Corp. 2022, 2023
```

```suggestion
 * Copyright IBM Corp. 2022, 2025
```

Can the last date in every copyright notice in the PR be 2025? First date is when the file was created, second date is the year in which the last edit was made.

---

Yep, just pushed this in the last commit, was leaving it to the end lol 👍